### DISCUSSION TITLE
Save settings to a file for importing on a reinstall

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/myMPD#784] [Ideas]
Save setting to a file for importing on a reinstall.  I've been working on a personal ```mpd-backup.sh``` shell script so that I can always revert to known working mpd setup files. It'd be nice to have the ability to save the current settings from the maintaince tab.

**Comment by jcorporation** [comment]
Settings are saved in single files in the directory `/var/lib/mympd/`. You can simply backup this files, e.g. with tar. I see no benefits of writing this data to a single file. Or I am missing something?

**Comment by bonelifer** [comment]
Thanks I wasn't sure where they were saved. So I just need to backup all of the ```/var/lib/mympd``` folder. Got it.
